airflow/contrib/operators/sagemaker_training_operator.py

```diff
@@ -29,23 +29,26 @@ class SageMakerTrainingOperator(SageMakerBaseOperator):
 
     This operator returns The ARN of the training job created in Amazon SageMaker.
 
-    :param config: The configuration necessary to start a training job (templated)
+    :param config: The configuration necessary to start a training job (templated).
+    
+        For details of the configuration parameter, See:
+        https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sagemaker.html#SageMaker.Client.create_training_job
     :type config: dict
     :param aws_conn_id: The AWS connection ID to use.
     :type aws_conn_id: str
-    :param wait_for_completion: if the operator should block until training job finishes
+    :param wait_for_completion: If wait is set to True, the time interval, in seconds,
+        that the operation waits to check the status of the training job.
     :type wait_for_completion: bool
     :param print_log: if the operator should print the cloudwatch log during training
     :type print_log: bool
     :param check_interval: if wait is set to be true, this is the time interval
         in seconds which the operator will check the status of the training job
     :type check_interval: int
-    :param max_ingestion_time: if wait is set to be true, the operator will fail
-        if the training job hasn't finish within the max_ingestion_time in seconds
-        (Caution: be careful to set this parameters because training can take very long)
-        Setting it to None implies no timeout.
+    :param max_ingestion_time: If wait is set to True, the operation fails if the training job
+        doesn't finish within max_ingestion_time seconds. If you set this parameter to None,
+        the operation does not timeout.
     :type max_ingestion_time: int
-    """
+    """  # noqa
```

Comment:
Was this comment needed? What was the Flake8 error without it?

Reply:
Just the external link is too long. I am not sure if there's a way to separate a long link to multiple lines.